instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/dynamodb.py

```diff
@@ -354,6 +374,12 @@ def __init__(self, call_context: _AwsSdkCallContext):
     def extract_attributes(self, attributes: _AttributeMapT):
         attributes[SpanAttributes.DB_SYSTEM] = DbSystemValues.DYNAMODB.value
         attributes[SpanAttributes.DB_OPERATION] = self._call_context.operation
+        attributes[SpanAttributes.DB_STATEMENT] = str(
+            self._call_context.params
+        )
+        attributes[
```

Comment:
`_DynamoDbExtension` is probably not the correct place to add the `db.statement` attribute, since this is tracing all methods that the [DynamoDb SDK](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client) provides, so even those that most likely do not require a `db.statement` attribute.

It probably would be better to add the attribute separately to the respective `_DynamoDbOperation`s that support it.